v1beta1/proto/grafeas.proto

```diff
@@ -566,24 +566,18 @@ message GetVulnerabilityOccurrencesSummaryRequest {
 
 // A summary of how many vulnerability occurrences there are per severity type.
 message VulnerabilityOccurrencesSummary {
-  // A map of how many occurrences were found for each severity.
-  repeated SeverityCount counts = 1;
   // A listing by resource of the number of fixable and total vulnerabilities.
-  repeated FixableTotalByDigest fixable_counts = 2;
-  // The number of occurrences for a specific severity.
-  message SeverityCount {
-    // The severity of the occurrences.
-    grafeas.v1beta1.vulnerability.Severity severity = 1;
-    // The number of occurrences with the severity.
-    int64 count = 2;
-  }
-  // Per resource counts of fixable and total vulnerabilites.
+  repeated FixableTotalByDigest fixable_counts = 1;
+
+  // Per resource and severity counts of fixable and total vulnerabilites.
   message FixableTotalByDigest {
```

Comment:
Probably should just rename this to "message Counts" in case we decide to change its breakdown later?

Reply:
I'd prefer to leave it more specific. If we change it later we'd need to version anyway.